services/print-pdf/_tools/xlsx_to_html_vet_patrol.py

```
@@ -0,0 +1,100 @@
+merge_skip: set[tuple[int, int]] = set()
+for mr in ws.merged_cells.ranges:
+    rs = mr.max_row - mr.min_row + 1
+    cs = mr.max_col - mr.min_col + 1
+    merge_master[(mr.min_row, mr.min_col)] = (rs, cs)
+    for r in range(mr.min_row, mr.max_row + 1):
+        for c in range(mr.min_col, mr.max_col + 1):
+            if (r, c) != (mr.min_row, mr.min_col):
+                merge_skip.add((r, c))
+
+
+def classify(value, r: int, c: int) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (10 > 6)

(PLR0911)